### PR TITLE
Updated knowledge base faq page

### LINKED ISSUE #17: Oppdatere FAQ side
**Link to the page of interest**

https://dhp-stottepakke.github.io/pages/difficult_faq

**Describe what you would change to the page**

Implementers forslag fra word-utkast inn hit

**List the tool(s)or resource(s) that are missing on the page (if applicable)**
NA

### LINKED ISSUE #97: Re-organize main
**Link to the page of interest**
https://dhp-stottepakke.github.io/pages/index

**Describe what you would change to the page**
Re-organize to more directly address target groups researchers and support staff.

**List the tool(s)or resource(s) that are missing on the page (if applicable)**
- Introduce 'About' section?

### PR DESCRIPTION
**Pull request by jennyostrop**
Closes #17
Closes #97

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,11 +17,11 @@
                 </p>
             </div>
         </section>
-		
+
 		<!-- deliverables -->
 		<section class="bg-light pb-5">
                   <div class="container">
-                      <h2 class="no-anchor text-center mb-3"> Resources </h2>
+                      <h2 class="no-anchor text-center mb-3"> Information about data management planning </h2>
```
Comment (korbinib) [inline index.html:24]:
```suggestion
                      <h2 class="no-anchor text-center mb-3"> Further information </h2>

```
The whole page is about RDM or not?

**Review by korbinib — COMMENTED** [review]
See suggestion

File: index.html (hunk 1 above)
Comment (korbinib) [inline index.html:24]:
```suggestion
                      <h2 class="no-anchor text-center mb-3"> About data management planning </h2>

```

**Review by korbinib — CHANGES_REQUESTED** [review]

**Review by korbinib — APPROVED** [review]

**Review by sveinho — COMMENTED** [review]
Look good

**Pull request merged**